### DISCUSSION TITLE
Подробная информация о предмете с возможностью перехода на дополнительные страницы

### DISCUSSION THREAD

**Discussion by 0niel** [0niel/university-app#226] [Идеи]
При клике на предмет в расписании можно открывать дополнительную страницу, в которой будет вся информация о предмете:
1. Название
2. Преподаватели
3. Аудитории
4. Время
5. Кусочек карты, на котором будет видно аудиторию

При клике на преподавателя, можно открывать страницу с его расписанием на неделю. При клике на аудиторию - страницу с расписанием предметов в аудитории на неделю. При клике на карту - маршрут до аудитории.

**Comment by Dragonprod** [comment]
Реализовано
